.github/ISSUE_TEMPLATE/moderator-offboarding.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "Moderator offboarding"
```

Comment:
We should plan to have an alumni section on the website btw.

Reply:
> We should plan to have an alumni section on the website btw.

Good idea, added in 38269f5d6b199fd1046c89d7c64338e15184607d and https://github.com/r-multiverse/r-multiverse.github.io/pull/40/commits/64809745d478ecffa19449f805c910d4dbe55d34.

For alumni, can we maintain a simple list with the name and GitHub profile page of each person? It seems reasonable to remove photos and bios.

Reply:
I would also mention the period they served. This might help to recognize people that contributed longer, and later on in 25 years to reconstruct a bit the history of r-multiverse.